src/git/printable.go

```diff
@@ -10,6 +10,22 @@ import (
 
 var noneString = "[none]"
 
+// getBranchAncestryRoots returns the branches with children and no parents
+func getBranchAncestryRoots() []string {
+	parentMap := Config().GetParentBranchMap()
+	roots := make([]string, len(parentMap))
+	i := 0
+	for _, parent := range parentMap {
+		if _, ok := parentMap[parent]; !ok && !util.DoesStringArrayContain(roots, parent) {
+			roots[i] = parent
```

Comment:
You could simplify this by defining `roots` as `[]string{}`, and using `roots = append(roots, parent)` here. Then you don't need `i` and you don't need to truncate `roots` on line 24.